src/orchestratord/src/controller/materialize/environmentd.rs

```diff
@@ -833,13 +833,29 @@ fn create_environmentd_statefulset_object(
             config
                 .bootstrap_builtin_catalog_server_cluster_replica_size
                 .as_ref()
-                .map(|size| {
-                    format!("--bootstrap-builtin-catalog-server-cluster-replica-size={size}")
-                }),
+                .map(|size| format!("--bootstrap-builtin-catalog-server-cluster-replica-size={size}")),
             config
                 .bootstrap_builtin_analytics_cluster_replica_size
                 .as_ref()
                 .map(|size| format!("--bootstrap-builtin-analytics-cluster-replica-size={size}")),
+            config
+                .bootstrap_builtin_system_cluster_replication_factor
+                .as_ref()
+                .map(|replication_factor| {
+                    format!("--bootstrap-builtin-system-cluster-replication-factor={replication_factor}")
+                }),
+            config
+                .bootstrap_builtin_probe_cluster_replication_factor
+                .as_ref()
+                .map(|replication_factor| format!("--bootstrap-builtin-probe-cluster-replication-factor={replication_factor}")),
+            config
+                .bootstrap_builtin_support_cluster_replication_factor
+                .as_ref()
+                .map(|replication_factor| format!("--bootstrap-builtin-support-cluster-replication-factor={replication_factor}")),
+            config
+                .bootstrap_builtin_analytics_cluster_replication_factor
+                .as_ref()
+                .map(|replication_factor| format!("--bootstrap-builtin-analytics-cluster-replication-factor={replication_factor}")),
```

Comment:
Missing a config for the catalog_server replication factor, maybe that's intentional? It seems like it would be nice to have parity for all system clusters here

Reply:
Per Justin:

> I think we should only include these options where it wouldn't be a complete footgun to set it to a value other than 1.

And I kinda agree with him here!